src/App.ts

```diff
@@ -21,11 +21,9 @@ export class App implements Service {
     constructor() {
         this.express = express();
         // add before route middleware's here
-        // @ts-ignore-next-line
-        this.express.use(morgan("short", {stream: morganLogger}));
+        this.express.use(morgan("short", {stream: morganLogger}) as RequestHandler);
         this.express.use(bodyParser.json());
-        // @ts-ignore-next-line
-        this.express.use(helmet());
+        this.express.use(helmet() as RequestHandler);
```

Comment:
https://stackoverflow.com/questions/56116660/how-to-use-helmet-js-from-typescript

Reply:
google morgan as well, similar solution

Reply:
I already did that but that doesn't work. If you want to try, feel free